srcpkgs/gcc/template

```diff
@@ -3,13 +3,13 @@
 # which use the version number.
 
 pkgname=gcc
-version=12.2.0
-revision=3
+version=13.2.0
+revision=1
 _minorver="${version%.*}"
 _majorver="${_minorver%.*}"
-_gmp_version=6.2.1
-_mpfr_version=4.1.0
-_mpc_version=1.2.1
+_gmp_version=6.3.0
+_mpfr_version=4.2.0
+_mpc_version=1.3.1
 _isl_version=0.24
```

Comment:
Maybe also update mpfr to 4.2.1 (#45725) and isl to 0.26 (#45951)?

Reply:
isl 0.25+ requires c++17 whereas gcc compiles with c++11.

Reply:
Is anyone really using gcc<7?

Reply:
The issue is that gcc uses -std=c++11 and isl gets compiled through gcc's makefile so the flag gets passed on.